### DISCUSSION TITLE
TOP INTERWIEW 150

### DISCUSSION THREAD

**Discussion by cagriciplak** [gusanmaz/bmb212-algorithms-2024#3] [Q&A]
top interwiew 150 kısmındaki linklere tıkladığımda bomboş sayfaya 
atıyor
![1](https://github.com/gusanmaz/bmb212-algorithms-2024/assets/165172903/ae1e19b1-30e5-494d-a3bd-e792659b6659)
![2](https://github.com/gusanmaz/bmb212-algorithms-2024/assets/165172903/ec3b2ace-776e-4dcf-824e-db8877bf0be3)

**Comment by gusanmaz** [comment]
Simdi test ettim, ben de bir sorun gozukmuyor. 
Sorun yasadigin linklerden birini gonderebilir misin?
